platform-support/openshift-route-service/deploy/service.yaml

```diff
@@ -14,6 +14,7 @@ spec:
       labels:
         run: openshift-route-service
     spec:
+      serviceAccountName: keptn-default
```

Comment:
are the service account privileges sufficient to create OpenShift routes?

Reply:
Thank you. I added a rule.